### ISSUE TITLE
Running a slightly modified version of the demo code results in an error

### ISSUE THREAD

**Issue by jhorowitz** [microsoft/presidio#590]
**Describe the bug**
Running into an AttributeError in what appears to be a trivial case. Perhaps I'm doing something wrong?

**To Reproduce**

[This colab reproduces the issue](https://colab.research.google.com/drive/1FmzrHkJV7uno5EIiFrQu6amNglVyx0Nj?usp=sharing) if the error below isn't enough.

Code:
```
text="My phone number is 212-555-5555 and the IBAN is IL150120690000003111111 "
results = analyzer.analyze(text=text, language='en')
anonymized_text = anonymizer.anonymize(text=text, analyzer_results=results)
```

Output:

```
AttributeError                            Traceback (most recent call last)
<ipython-input-4-834d0ad09e29> in <module>()
      1 text="My phone number is 212-555-5555 and the IBAN is IL150120690000003111111 "
      2 results = analyzer.analyze(text=text, language='en')
----> 3 anonymized_text = anonymizer.anonymize(text=text, analyzer_results=results)

3 frames
/usr/local/lib/python3.7/dist-packages/presidio_anonymizer/entities/analyzer_results.py in <listcomp>(.0)
     61         for result_a in self:
     62             other_elements.remove(result_a)
---> 63             if not any([result_a.has_conflict(other_element) for other_element in
     64                         other_elements]):
     65                 other_elements.append(result_a)

AttributeError: 'RecognizerResult' object has no attribute 'has_conflict'
```

**Comment by hkarakose** [comment]
I have the problem too. Can we rollback to a previous version to get rid of the problem until bug fix has been released?

**Comment by omri374** [comment]
Hi, we are working on a fix. A workaround: 

```python
from presidio_analyzer import AnalyzerEngine
from presidio_anonymizer import AnonymizerEngine
from presidio_anonymizer.entities import AnalyzerResult

analyzer = AnalyzerEngine()
anonymizer = AnonymizerEngine()


text="My phone number is 212-555-5555 and the IBAN is IL150120690000003111111 "
results = analyzer.analyze(text=text, language='en')

# translate results to fit the anonymizer (this will be unecessary on the next version)
new_results = [AnalyzerResult(start=r.start, end=r.end, entity_type=r.entity_type, score=r.score) for r in results]

anonymized_text = anonymizer.anonymize(text=text, analyzer_results=new_results)
```

This is just a workaround until we fix the issue.

**Comment by jhorowitz** [comment]
I've tested the workaround and can confirm it works as expected

**Comment by omri374** [comment]
Great. We already have a fix on the main branch which will be released today.

**Comment by omri374** [comment]
Fixed in version 2.0.1

**Comment by jhorowitz** [comment]
That was so fast! Thank you very much
